webapp/src/components/workspace.tsx

```diff
@@ -36,6 +36,7 @@ function CenterContent(props: Props) {
     const groupByProperty = useAppSelector(getCurrentViewGroupBy)
     const dateDisplayProperty = useAppSelector(getCurrentViewDisplayBy)
     const clientConfig = useAppSelector(getClientConfig)
+    const hiddenCards = useAppSelector(GetCurrentBoardHiddenCards)
```

Comment:
We should wrap this with the normal Cloud checks. `utils.IsCloudLicense() and GetCloudLimits`

This will slow down on-premise and paid accounts. by looping through cards unnecessarily.